client/packages/requisitions/src/RnRForms/ListView/RnRFormCreateModal.tsx

```diff
@@ -166,8 +166,20 @@ const PeriodSelect = ({
   previousFormExists?: boolean;
   errorMessage?: string;
 }) => {
+  const t = useTranslation('replenishment');
+
   return (
     <Box display="flex" flexDirection="column">
+      <Typography
+        sx={{
+          fontStyle: 'italic',
+          color: 'gray.main',
+          fontSize: '12px',
+          width,
+        }}
+      >
+        {t('messages.only-closed-periods-visible')}
```

Comment:
Haha sorry one more side quest, this was a request to add some explanation as to why I could have more periods configured in mSupply but they don't show up here... didn't really feel worth its own issue 🙈